.github/workflows/e2e.yml

```diff
@@ -104,14 +111,33 @@ jobs:
 
       - name: Run screenshot tests on ${{ env.BASE_REF_DEMO }}
         continue-on-error: true
-        env: # note: need to collect all
-          # diff files for lumberjack bot
-          CYPRESS_failOnSnapshotDiff: 0
         run: npx nx e2e demo-integrations --key next --folder ${{ matrix.project }}
 
       - name: Clean up resources
         run: npx kill-port --port ${{ env.NG_SERVER_PORT }}
 
+      - name: Update artifacts / ${{ env.SNAPSHOTS_KEY }}
+        uses: actions/upload-artifact@v3
```

Comment:
Do you know about limits for free organization?

Reply:
Yes, around 10Gb for all agents. Meanwhile, now we do not experience problems in this, since screenshots are expired in a day. In this PR, I simply just moved this code to another the job.